src/model/topic.ts

```diff
@@ -42,3 +47,13 @@ export type Topic = Owned & {
   uri: string
   spatial: SpatialField | undefined
 }
+
+export type TopicPostData = Omit<
```

Comment:
J'ai du mal à comprendre le déroulé autour de ce type.
C'est parce que dans le cas normal le store fait les conversions et dans le cas d'un clone on le fait à la mano dans `cloneTopic` ?

Reply:
J'ai dû introduire ce type parce que le passage par une fonction entraine plus de contraintes sur le typage et qu'on va plus loin dans le clone en faisait un reset de certains attributs. Avant on s'en sortait simplement avec un `Partial<Topic>` qui était hydraté avec ce dont on avait besoin pour le POST. Maintenant je dois construire explicitement un dataset cloné au format v1 via `updateClone`. C'est notamment le travail sur `datasets` and `reuses` qui fait la différence.